lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -358,7 +379,7 @@ rule CONNECTORType2Object {
 		className <- 'LPCode.LinkClass',
 		number <- thisModule.linkClassNumberMap.get(thisModule.getConnectorTargetId(c)),
```

Comment:
Why don't you use also a shortcut here, like the `getNumber()` a few lines below?